packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -52,7 +52,7 @@ const getOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedPa
     previousOverflowParticipants = []
   } = props;
 
-  const callingParticipants = remoteParticipants.filter((p) => p.state === ('Connecting' || 'Ringing'));
+  const callingParticipants = remoteParticipants.filter((p) => p.state === 'Connecting' || p.state === 'Ringing');
```

Comment:
@dmceachernmsft please review this closely as this is a functional change in behavior

Reply:
Removed from this PR as it is a functional change causing many snapshot changes and needs to be its own PR. Have started offline chat about this